app/models/citysdk/request_filter.rb

```diff
@@ -102,12 +102,12 @@ def filter_observation_key(params)
     def filter_area_code(params)
       @collection = @collection.where(<<~SQL.squish)
         ST_Within(#{Issue.quoted_table_name}."position",
-          (#{Authority.where(id: params[:area_code].to_i).select(:area).to_sql}))
+          (#{District.where(id: params[:area_code].to_i).select(:area).to_sql}))
```

Comment:
Gibt es mit dem Wechsel von Authority zu District keine Probleme mit der MV Instanz?

Reply:
Nicht bei der Statistik. Die Probleme in der Vergangenheit waren bei den Beobachtungsflächen. Dies hat hierdurch aber keinen Einfluss.